package.json

```diff
@@ -7,23 +7,5 @@
     "vue-axios": "2.1.4",
     "vue-router": "3.0.6",
     "vuelidate": "0.7.4"
-  },
-  "devDependencies": {
-    "fs": "0.0.1-security",
-    "gulp": "4.0.2",
-    "gulp-all": "1.1.0",
-    "gulp-clean-css": "4.3.0",
-    "gulp-plumber": "1.2.1",
-    "gulp-rename": "2.0.0",
-    "gulp-rollup": "2.17.0",
-    "gulp-sass": "4.1.0",
-    "gulp-uglify": "3.0.2",
-    "node-sass": "5.0.0",
-    "rollup-plugin-alias": "1.5.1",
-    "rollup-plugin-buble": "0.19.6",
-    "rollup-plugin-commonjs": "8.4.1",
-    "rollup-plugin-json": "4.0.0",
-    "rollup-plugin-node-resolve": "4.1.0",
-    "rollup-plugin-replace": "2.2.0"
```

Comment:
Please look at https://github.com/Lombiq/Orchard-Vue.js/blob/dev-orchard-1/Assets/Scripts/helpers/get-vue-app-compiler-pipeline.js - there, these dependencies are used.

Reply:
And that's sadly used in exactly one place. Yeah, thanks.